File: iniciante/1005/1005.py
# Lê um número decimal (float) do usuário e armazena na variável 'a'
a = float(input())

# Lê outro número decimal (float) do usuário e armazena na variável 'b'
b = float(input())

# Define os pesos das notas 'a' e 'b'
peso_a = 3.5
peso_b = 7.5

# Calcula a média ponderada considerando os pesos e as notas
media = ((peso_a * a) + (peso_b * b)) / (peso_a + peso_b)

# Imprime a média com 5 casas decimais, no formato "MEDIA = valor"
print(f"MEDIA = {media:.5f}")

File: iniciante/1005/1005.c
// Inclui a biblioteca padrão de entrada e saída
#include <stdio.h>

int main() // Função principal do programa
{
  double a, b, media; // Declara variáveis do tipo double para armazenar as notas e a média

  // Lê o primeiro número decimal (double) digitado pelo usuário e armazena em 'a'
  scanf("%lf", &a);

  // Lê o segundo número decimal (double) digitado pelo usuário e armazena em 'b'
  scanf("%lf", &b);

  // Calcula a média ponderada com pesos 3.5 para 'a' e 7.5 para 'b'
  media = ((a * 3.5) + (b * 7.5)) / 11;

  // Imprime a média com 5 casas decimais, no formato "MEDIA = valor"
  printf("MEDIA = %.5lf\n", media);

  return 0; // Indica que o programa terminou com sucesso
}
